src/app/product-details-page/product-details-page.module.ts

```diff
@@ -4,9 +4,10 @@ import { CommonModule } from '@angular/common';
 import { ProductDetailsPageRoutingModule } from './product-details-page-routing.module';
 import { ProductDescriptionComponent } from './components/product-description/product-description.component';
 import { ProductDetailsPageComponent } from './components/product-details-page/product-details-page.component';
+import { ZoomitComponent } from './components/product-description/product-image-preview/zoomit/zoomit.component';
```

Comment:
what's the weird name for the component, zoomit? rename to Zoom or ZoomImage Component